README.md

```diff
@@ -227,6 +227,21 @@ It also supports an additional `rewriteRequestHeaders(headers, request)` functio
 opening the WebSocket connection. This function should return an object with the given headers.
 The default implementation forwards the `cookie` header.
 
+## `wsReconnect`
+
+The `wsReconnect` option contains the configuration for the WebSocket reconnection feature; is an object with the following properties:
```

Comment:
We need an `async onReconnect(oldSocket, newSocket)` or some other way to carry state from the old connection to the new connection.